go.mod

```diff
@@ -50,15 +50,15 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/replicatedhq/kurl v0.0.0-20210414162418-8d6211901244
-	github.com/replicatedhq/troubleshoot v0.38.0
+	github.com/replicatedhq/troubleshoot v0.38.1-0.20220727004344-14f9ae8f3b26
```

Comment:
This version is on a branch.  Need to update this once the corresponding PR in troubleshoot is merged.